backend/typescript/types.ts

```diff
@@ -241,7 +241,9 @@ export type UpdateCamperDTO = Omit<
   | "latePickup"
   | "contacts"
   | "optionalClauses"
->;
+  | "campSession"
+> &
+  Partial<Pick<CamperDTO, "campSession">>;
```

Comment:
Similar comment to below, why is this needed?

Reply:
Good point. I'll remove lines 244, 245, and 246. the reason I included it was because I though that having `UpdateCamperDTO.campSession` be required might lead to runtime errors when `UpdateCamperDTO.campSession` isn't provided (as in the case of edit camper modal). But I guess it's just filled in to be undefined by some other code.